docs/FAQ/FAQ.qmd

```diff
@@ -27,7 +27,32 @@ eatRep::pool.R2(r2)
 
 ## 2. DIF-Richtung in Conquest/eatModel: Was bedeuten negative Werte in der "estDif"-Spalte?
 
-In eatModel wird nur der erste Teil der zweiten Tabelle item\*\[DIFvariable\] aus Conquest übertragen. Das bedeutet, dass negative Werte in der eatModel-estDif-Spalte dafür stehen, dass die jeweiligen Items **in der ersten Gruppe** der \[DIFvariable\] **leichter** sind. Die erste Gruppe ist die mit dem numerisch kleineren Gruppenindikatorwert.
+In eatModel wird nur der erste Teil der zweiten Tabelle $item\times [DIFvariable]$ aus Conquest übertragen. Das bedeutet, dass negative Werte in der eatModel-estDif-Spalte dafür stehen, dass die jeweiligen Items **in der ersten Gruppe** der \[DIFvariable\] **leichter** sind. Die erste Gruppe ist die mit dem numerisch kleineren Gruppenindikatorwert. Zur Illustration ein Beispiel, das Daten aus dem TAM-Paket verwendet:
+
+```{r eval = TRUE}
```

Comment:
please add warning=FALSE,message=FALSE --> {r eval = TRUE,warning=FALSE,message=FALSE}
in order to avoid unnecessary spam on the site (eatModel core messages will be displayed despite these settings)

Reply:
I've changed this.